.github/workflows/committed-chart.yaml

```diff
@@ -0,0 +1,33 @@
+name: Chart Generation
```

Comment:
```suggestion
name: Check chart generation
```

Reply:
How about renaming this as well as the name of the workflow file to be a bit more descriptive? We're just doing a check between the chart generated against the current state of the main branch versus the PR branch.